ddsrecorder/src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -33,7 +33,7 @@ namespace eprosima {
 namespace ddsrecorder {
 namespace recorder {
 
-using LogFilter = std::map<eprosima::fastdds::dds::Log::Kind, std::string>;
+using LogFilter = std::map<fastdds::dds::Log::Kind, std::string>;
```

Comment:
It's not necessary to reimplement `LogFilter`. Please use the one defined in dev-utils.